integration-tests/src/main/java/org/apache/druid/testing/utils/KinesisAdminClient.java

```diff
@@ -156,15 +162,16 @@ public boolean verfiyPartitionCountUpdated(String streamName, int oldShardCount,
     return actualShardCount == oldShardCount + newShardCount;
   }
 
-
-  private boolean verifyStreamStatus(StreamDescription streamDescription, StreamStatus streamStatusToCheck)
+  private List<Shard> listShards(String streamName)
   {
-    return streamStatusToCheck.toString().equals(streamDescription.getStreamStatus());
```

Comment:
Can we add a comment with the aws tps limits currently for the `listshards` call ?